src/LabelsExtension.php

```diff
@@ -81,12 +81,14 @@ protected function registerBackendControllers()
      */
     protected function registerMenuEntries()
     {
-        return [
-            (new MenuEntry('labels', 'labels'))
-                ->setLabel('Label translations')
-                ->setIcon('fa:flag')
-                ->setPermission('labels'),
-        ];
+        if ($app['labels.config']->isShowMenu()) {
+            return [
+                (new MenuEntry('labels', 'labels'))
+                    ->setLabel('Label translations')
+                    ->setIcon('fa:flag')
+                    ->setPermission('labels'),
+            ];
+        }
     }
```

Comment:
This function needs to always [return an array](https://github.com/bolt/bolt/blob/release/3.2/src/Extension/MenuTrait.php#L26) plus you're better off overall not nesting, e.g.
```
    protected function registerMenuEntries()
    {
        if (!$app['labels.config']->isShowMenu()) {
            return [];
        }

        return [
            (new MenuEntry('labels', 'labels'))
                 ->setLabel('Label translations')
                 ->setIcon('fa:flag')
                 ->setPermission('labels'),
        ];
    }
```